### ISSUE TITLE
Output of chebfun.prod

### ISSUE THREAD

**Issue by asgeirbirkis** [chebfun/chebfun#1436]
Currently, ```chebfun.prod``` returns a ```chebfun``` of length 1. Shouldn't it really just return a scalar? Cf. the help text
```
>> help chebfun/prod
 prod   Product integral.
    prod(F) is the definite product integral of the CHEBFUN F, which is defined
    as exp(sum(log(F))).
```
And the following calculations that to me don't make much sense
```
>> f = chebfun(@(x) sin(x) + 2);
>> exp(sum(log(f)))
ans =
    3.7205
>> prod(f)
ans =
   chebfun column (1 smooth piece)
       interval       length   endpoint values  
[      -1,       1]        1       3.7      3.7 
Epslevel = 1.193616e-15.  Vscale = 3.720538e+00.
```

**Comment by nickhale** [comment]
Yes.

**Comment by aaustin141** [comment]
Closed in 7d9aec93 from #1596.
